scripts/seed-service.ts

```diff
@@ -24,3 +25,49 @@ export const seedUsers = async (length?: number): Promise<User[]> => {
 
   return createdUsers;
 };
+
+export const seedUsersWithAddress = async (length?: number): Promise<UserType[]> => {
```

Comment:
```suggestion
export const seedUsersWithAddress = async (length: number = 50): Promise<UserType[]> => {
```
com isso não precisa do if da linha 30, uma vantagem desse tipo de abordagem é tratar as variáveis como constante, o que acaba no longo prazo prevenindo efeitos colaterais (acho isso um principio legal da programação funcional e que vale a pena aplicar no js)